gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -396,26 +400,48 @@ void ROOT::Experimental::TWebWindowsManager::TestProg(TString &prog, const std::
 ///               $height - widget height
 ///
 ///  If allowed, same window can be displayed several times (like for TCanvas)
-
-bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow &win, const std::string &_where)
+///  Following parameters can be configured in rootrc file:
+///
+///   WebGui.Chrome:  full path to Google Chrome executable
+///   WebGui.ChromeBatch: command to start chrome in batch
+///   WebGui.ChromeInteractive: command to start chrome in interactive mode
+///   WebGui.Firefox: full path to Mozialla Firefox executable
+///   WebGui.FirefoxBatch: command to start Firefox in batch mode
+///   WebGui.FirefoxInteractive: command to start Firefox in interactive mode
+///   WebGui.FirefoxProfile: name of Firefox profile to use
+///   WebGui.FirefoxProfilePath: file path to Firefox profile
+///   WebGui.FirefoxRandomProfile: usage of random Firefox profile -1 never, 0 - only for batch mode (dflt), 1 - always
+///   WebGui.LaunchTmout: time required to start process in seconds (default 30 s)
+///   WebGui.OperationTmout: time required to perform WebWindow operation like execute command or update drawings
+
+unsigned ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow &win, bool batch_mode, const std::string &_where)
```

Comment:
This function has 360 lines. Please consider splitting it into smaller functional parts.

Reply:
Yes, It is large, but I prefer to keep it central. At least for now,